src/Components/ADT3DSceneBuilder/Internal/Shared/SharedFormUtils.tsx

```diff
@@ -0,0 +1,39 @@
+import { Icon, IDropdownOption } from '@fluentui/react';
+import React from 'react';
+
+const iconStyles = { marginRight: '8px' };
+const optionWrapperStyle = { display: 'flex', alignItems: 'center' };
+
+export const onRenderTypeOption = (option: IDropdownOption): JSX.Element => {
+    return (
+        <div style={optionWrapperStyle}>
+            {option.data && option.data.icon && (
+                <Icon
+                    style={iconStyles}
+                    iconName={option.data.icon}
+                    aria-hidden="true"
+                    title={option.data.icon}
+                />
+            )}
+            <span>{option.text}</span>
+        </div>
+    );
+};
+
+export const onRenderTypeTitle = (options: IDropdownOption[]): JSX.Element => {
```

Comment:
ummmm. Do we need this function? Or can we just use the other one? Cause it seems like they are just a copy. Could we just call the other function instead and pass in the `options[0]` as the prop?

Reply:
These are being called by Dropdown component, I think it is cleaner to have 2 functions instead of having one and calling it inline in the dropdown props and passing `options[0]`.

i.e.
``` 
<Dropdown
    ...
    onRenderTitle={(options) => {  onRenderTypeOption(options[0[) }}
/>
```

Reply:
I'm fine if you want the 2 utilties. But since we want them to always match, can't you juts call the other utility from here? grab your single option then `return myOtherUtility(option);`

Reply:
Ah I see what you are saying, let me make that change